web/admin/application/configs/klear/model/KamRtpengine.yaml

```diff
@@ -0,0 +1,58 @@
+production:
+  entity: Ivoz\Kam\Domain\Model\Rtpengine\Rtpengine
+  fields: 
+    url:
+      title: ngettext('URL', 'URLs', 1)
+      type: text
+      trim: both
+      required: true
+      maxLength: 128
+      info:
+        type: box
+        position: left
+        icon: help
+        text: _("e.g. udp:127.0.0.1:22223")
+        label: _("Need help?")
+    weight:
+      title: _('Weight')
+      type: number
+      defaultValue: 1
+      source:
+        control: Spinner
+        min: 0
+        max: 254
+      info:
+        type: box
+        position: left
+        icon: help
+        text: _("0 disables if more available, 2 gets double calls than 1, 3 gets triple calls than 1, etc.")
+        label: _("Need help?")
+    description:
+      title: _('Description')
+      type: text
+      trim: both
+      maxLength: 200
+    disabled:
+      title: _('Description')
```

Comment:
I think this disabled field is decription duplicated with the title changed. 

It is also overwritten with the next field definition because they share the name